src/main/java/uk/gov/hmcts/reform/slc/services/SendLetterClient.java

```diff
@@ -109,13 +119,21 @@ public Health serviceHealthy() {
         }
     }
 
-    private void restTemplatePut(String url, Object body) {
+    private void callRestTemplate(String url, HttpMethod method, Object body) {
         HttpHeaders headers = new HttpHeaders();
         headers.add(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_UTF8_VALUE);
         headers.add(AUTHORIZATION_HEADER, authTokenGenerator.generate());
 
         HttpEntity<Object> entity = new HttpEntity<>(body, headers);
 
-        restTemplate.exchange(url, HttpMethod.PUT, entity, Void.class);
+        restTemplate.exchange(url, method, entity, Void.class);
+    }
+
+    private void restTemplatePost(String url, Object body) {
```

Comment:
You can get rid of these methods `restTemplatePut` and `restTemplatePost` and just call `callRestTemplate(..)`.What do you think?

Reply:
didn't want to change existing 3 methods. i can remove object from the post since it is a single call with `null`